crates/astria-conductor/build.rs

```diff
@@ -0,0 +1,4 @@
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    astria_build_info::emit("conductor-v")?;
```

Comment:
Should we just use `env!("CARGO_PKG_NAME")`?

Reply:
We are using `astria-conductor` for pkg name but `conductor-vx.x.x` for the tag so that doesn't work unfortunately.